api/src/main/java/org/eclipse/microprofile/lra/annotation/ws/rs/LRA.java

```diff
@@ -61,6 +61,13 @@
  *   <li>to cancel the LRA context when the method returns particular HTTP
  *       status codes</li>
  * </ul>
+ * 
+ * Some of the above defined LRA operations are performed before the execution
+ * proceeds to the LRA annotated business method. If any of these LRA operations
+ * cannot complete the implementation MUST return one of the defined HTTP status
+ * codes which means that the business method is not invoked. For the definition of
+ * the allowed  * status codes and condition in which they are returned please
```

Comment:
is this a typo or I don't understand the meaning? I mean the `the allowed  * status`

Reply:
typo, thanks @ochaloup